Sparkle/SUAutomaticUpdateAlert.m

```diff
@@ -100,4 +107,16 @@ - (NSString *)descriptionText
     }
 }
 
+- (NSTouchBar *)makeTouchBar
+{
+    self.touchBarProvider = [[SUTouchBarProvider alloc] initWithIdentifier:self.className];
+    [self.touchBarProvider addButtonWithButton:self.skipButton isDefault:NO];
+    [self.touchBarProvider addSpace];
+    [self.touchBarProvider addButtonWithButton:self.laterButton isDefault:NO];
+    [self.touchBarProvider addButtonWithButton:self.installButton isDefault:YES];
+    [self.touchBarProvider addSpace];
+    [self.touchBarProvider addSpace];
```

Comment:
I don't think two spaces as extra padding should be added. It seems kind of hacky/maybe fragile to me, and I think it looks visually fine without them.